components/brave_new_tab_ui/components/search/SearchDialog.tsx

```diff
@@ -17,6 +17,7 @@ interface Props {
 
 const duration = '0.4s'
 const easing = 'cubic-bezier(0.7, -0.4, 0.4, 1.4)'
+const fillMode = 'both'
```

Comment:
Adding a fill mode ensures that we don't have flash of the search dialog at its default position when closing it (the flash was noticeable on my dev build).